plugin/bullets.vim

```diff
@@ -1089,9 +1075,10 @@ fun! s:get_visual_selection_lines()
   let l:index = l:lnum1
   let l:lines_with_index = []
   for l:line in l:lines
-    let l:lines_with_index += [{'text': l:line, 'nr': l:index}]
```

Comment:
This is an optimisation.

However, the entire `get_visual_selection_lines()` is not needed anymore and perhaps ought to be removed.